src/app/accounts/add-earn.ts

```diff
@@ -76,7 +76,12 @@ export const addEarn = async ({
   const shouldGiveReward = await RewardsRepository(accountId).add(quizQuestionId)
   if (shouldGiveReward instanceof Error) return shouldGiveReward
 
-  const payment = await Payments.intraledgerPaymentSendWalletId({
+  const intraledgerPaymentSendFn =
```

Comment:
my assumption was that funder wallet would always be a Btc wallet but I guess you want to remove this assumption?

Reply:
No strong opinions on my end. I just wasn't sure what the assumption was. Fixed here: https://github.com/GaloyMoney/galoy/pull/2105/commits/7dadb9ad6e3e9cf0426612db7fe1de8d1c35ce78